io.github.maddecoder.Classic-RBDOOM-3-BFG.yml

```diff
@@ -0,0 +1,139 @@
+app-id: io.github.maddecoder.Classic-RBDOOM-3-BFG
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "22.08"
+command: DOOMBFA.sh
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.mono6
+build-options:
+    append-path: /usr/lib/sdk/mono6/bin
+    append-ld-library-path: /usr/lib/sdk/mono6/lib
+    append-pkg-config-path: /usr/lib/sdk/mono6/lib/pkgconfig
+
+finish-args:
+- --device=all
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+- --share=network
+- --socket=pulseaudio
+- --filesystem=~/.var/app:ro
+- --filesystem=home
+- --filesystem=/run/media:ro
+- --filesystem=/media:ro
```

Comment:
these are very broad filesystem permissions.

Why do you need `.var/app`?

Reply:
They are broad because they are used to lookup for where the game is installed (putting static paths can lead to many errors).

Reply:
isn't the game installed in the flatpak?

Reply:
this doesn't justify `~/.var/app`

Reply:
This is just a source port for the game. The end user must have the game installed separately.
The ~/.var/app is for game installers running under flatpak

Reply:
so what's that big extra-data blob? you download?

Reply:
Additional assets the engine uses. Due to the .WAD files contain possibly some copyrighted assets, I put them on extra-data

Reply:
Is the main game available as flatpak? What is the app id?

Reply:
Also why do you need full home? Can `persist=.doombfa` replace it?

Reply:
> The ~/.var/app is for game installers running under flatpak

Do these have a specific flatpak id or are they on flathub?

You can then restrict it to that subdirectory only.

Reply:
> Do these have a specific flatpak id or are they on flathub?

It's mainly for Steam, Heroic Launcher and Lutris. I guess I can change it to these 3 apps instead

Reply:
Ok, I limit it to those 3 apps

Reply:
For the others sadly they are wild card paths (aka the user can install the game on any custom path)